src/sidenav/components/SidenavMenu.vue

```diff
@@ -32,6 +32,39 @@
         </QBadge>
       </QItemSection>
     </QItem>
+    <QItem
```

Comment:
This looks like leftovers from a previous PR.
You can restore it to the previous version: `git checkout master -- src/sidenav/components/SidenavMenu.vue`